src/_data/companies.yml

```diff
@@ -339,3 +339,11 @@ Loxysoft:
   github: https://github.com/rbkmoney
   location: Moscow, Russia
   description: Erlang is a core of our payment platform serving millions of customers around the world.
+  
+ "Peer Stritzinger GmbH":
```

Comment:
Oh, I see there is an indentation problem here, could you fix this up?

Reply:
Hopefully fixed, Github online editor had its own ideas how to indent

Reply:
LGTM!